op-interop-mon/monitor/job_test.go

```diff
@@ -124,3 +126,37 @@ func TestJobFromLog(t *testing.T) {
 		})
 	}
 }
+
+func TestJobId(t *testing.T) {
```

Comment:
Can we add a test for this where two identical receipts create two identical IDs, and then we change one of each:
- log index
- identifier info
- executing chain
- executing block
- etc

and observe that the two IDs never equal one another when any of these values differ?